style.css

```diff
@@ -221,7 +248,7 @@ header {
 }
 
 .home-text {
-	background-color: rgba(255, 255, 255, 0.178);
+	background-color: var(--color-transparent-white);
 	padding: 30px;
```

Comment:
Can we replace these padding values on the form? A few more below.

Reply:
got these changed!